src/plotting/plotList.component.tsx

```diff
@@ -38,6 +38,7 @@ const PlotCard = (props: { plotTitle: string; plotConfig: PlotConfig }) => {
   );
 };
 
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
 interface PlotListProps {}
```

Comment:
Why do we have this here again?

Reply:
It's just a standard thing - makes it easier to add new props this way. Although I'm not sure whether we'd ever have to pass props to this, so perhaps I should just delete it